enhancer.test.js

```diff
@@ -0,0 +1,52 @@
+const e = require('./enhancer');
+const testItems = require('./testItems');
+
+const items = testItems.items;
+const levels = testItems.levels;
+
+describe('the enhancer object', () => {
+  describe('the success function', () => {
+    test('returns a new item with its enhacement increased by 1', () => {
+      expect(e.success(items[0]).enhancement).toEqual('+1');
+      expect(e.success(items[1]).enhancement).toEqual('PEN');
+      expect(e.success(items[2]).enhancement).toEqual('PRI');
+    });
+    test('return new item with the new enhacement level reflected in the name', () => {
+      expect(e.success(items[0]).name).toEqual('[+1] Elvan Sword');
+      expect(e.success(items[1]).name).toEqual('[PEN] Elvan Shield');
+      expect(e.success(items[2]).name).toEqual('[PRI] Long Bow');
+    });
+  });
+
+  describe('the fail function', () => {
+    test('returns the durability of an item decreased by 5, but not lower than 20, if enhancement is between +0 and +14', () => {
+      expect(e.fail(items[0]).durability).toEqual(75); // enhancement level entering function: +0
```

Comment:
Good use of comments on each test Jesse. By labeling the tests in this way you're giving a reader who may not be familiar with the rest of the codebase, some needed context 👍